helm/opendistro-es/templates/configurer/configurer-config.yml

```diff
@@ -0,0 +1,31 @@
+{{- if .Values.configurer.enabled }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "opendistro-es.fullname" . }}-configurer-config
+  labels:
+{{ include "opendistro-es.labels.standard" . | indent 4 }}
+data:
+  configurer.sh: |-
+{{ tpl (.Files.Get "files/configurer/configurer.sh") . | indent 4 }}
+
+  kibana-dashboards.ndjson: {{ .Files.Get "files/kibana-resources/kibana-resources.ndjson" | quote }}
+
+  kubeaudit_template.json: |-
+{{ tpl (.Files.Get "files/configurer/kubeaudit_template.json") . | indent 4 }}
+
+  kubernetes_template.json: |-
+{{ tpl (.Files.Get "files/configurer/kubernetes_template.json") . | indent 4 }}
+
+  other_template.json: |-
+{{ tpl (.Files.Get "files/configurer/other_template.json") . | indent 4 }}
```

Comment:
These are not really templated, maybe skipt the `tpl`? Alternatively, make a single template that can be used for all of them. :slightly_smiling_face: